backend/src/components/edx/exchange.js

```diff
@@ -724,6 +724,26 @@ async function removeSecureExchangeStudent(req, res){
   }
 }
 
+async function createSecureExchangeNote(req, res) {
+  try {
+    const token = utils.getBackendToken(req);
+    const userInfo = utils.getUser(req);
+    const message = req.body;
+    const payload = {
+      secureExchangeID: req.params.secureExchangeID,
+      staffUserIdentifier: userInfo.idir_username,
+      content: message.content,
+      noteTimestamp: LocalDateTime.now().format(DateTimeFormatter.ofPattern('yyyy-MM-dd\'T\'HH:mm:ss')),
+    };
+
```

Comment:
Please add check for staff permission here (SECURE_EXCHANGE). See the removeStudent method for the example you can steal.

Reply:
Added security check in later commit.